libs/datasets/sources/covid_tracking_source.py

```diff
@@ -131,6 +132,62 @@ def standardize_data(cls, data: pd.DataFrame) -> pd.DataFrame:
             == data[cls.Fields.TOTAL_TEST_RESULTS_INCREASE]
         ).all()
 
+        nevada_data = cls._load_nevada_override_data()
+        data = cls._add_nevada_data(data, nevada_data)
+
         # TODO implement assertion to check for shift, as sliced by geo
         # df['totalTestResults'] - df['totalTestResultsIncrease']  ==  df['totalTestResults'].shift(-1)
-        return data
\ No newline at end of file
+        return data
+
+    @classmethod
+    def _load_nevada_override_data(cls):
+        from libs.datasets import NevadaHospitalAssociationData
+
+        data = NevadaHospitalAssociationData.local().timeseries(fill_na=False).data
+        columns_to_include = []
+        for timeseries_column, ct_column in cls.TIMESERIES_FIELD_MAP.items():
+            if timeseries_column in data.columns:
+                columns_to_include.append(ct_column)
```

Comment:
Does this mean that NHA data will only replace Covid Tracking data for the dates we have NHA data for? Will that cause any inconsistency day-to-day?

Reply:
Correct.  

For right now the model seems to be using the latest values, so this is actually okay as long as covid tracking doesn't start reporting nv data.  Probably worth an issue to make sure this doesn't bite us in the future though.